README.adoc

```diff
@@ -35,6 +36,33 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+Using npm:
+
+ $ npm i -g npm
+ $ npm i --save asciidoctor.js
+ $ npm i --save asciidoctor-reveal.js
+
+In Node.js:
+
+[source, javascript]
+----
+// Load asciidoctor.js + asciidoctor-template.js
+var asciidoctor = require('asciidoctor.js')();
+var Asciidoctor = asciidoctor.Asciidoctor();
+var Opal = asciidoctor.Opal;
+Opal.load('nodejs');
+Opal.load('pathname');
+require('asciidoctor-template.js');
+
+// Convert the document 'presentation.adoc' using Reveal.js backend
+var options = Opal.hash({safe: 'safe',
+                         template_dir: 'node_modules/asciidoctor-reveal.js/templates',
+                         backend: 'revealjs'});
+var result = Asciidoctor.$convert_file('presentation.adoc', options);
```

Comment:
The fact that `presentation.adoc` is a filename and that it is related to what you are trying to render is important and should be clarified in the documentation outside the code listing.